app/logging.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import absolute_import
```

Comment:
This one we do need because we're making use of the standard library logging package that is the same name as ours.